app/lib/common/providers/common_providers.dart

```diff
@@ -67,6 +79,8 @@ class EmailAddresses {
 
 final emailAddressesProvider = FutureProvider((ref) async {
   final client = ref.watch(alwaysClientProvider);
+  // ensure we are updated if the upgrade comes down the wire.
+  ref.watch(genericUpdatesStream('global.acter.dev.three_pid'));
```

Comment:
What is `global.acter.dev.three_pid` ? Is that matrix third-party id from acter home server?

Reply:
No, it's the internal event name for where we store the email address. This isn't super great like this but something we can tackle once we have Emil support by dart-uniffi